ustriage/task.py

```diff
@@ -190,7 +191,7 @@ def compose_dup(self, shortlinks=True, extended=False):
 
         flags = self.get_flags()
 
-        text = '%s - %3s %-13s %-19s' % (
+        text = '%s - %4s %-13s %-19s' % (
```

Comment:
Since get_flags() now returns a fixed length string, it's unnecessary to specify the field width here, and will be more maintainable if it doesn't.  I.e., you just need:

`         text = '%s - %s %-13s %-19s' % (`

both here and above.

Reply:
Agreed, and I updated the docstring of the `get_flags` to outline that it has to be fixed-length.